CONTRIBUTING.md

```diff
@@ -24,33 +24,73 @@ The ILP RFC process attempts to be easy to use but also rigorous enough that the
 
 The process differentiates between specifications that are incubating (similar to IETF Internet Drafts) and those that are mature (similar to IETF RFCs).
 
-The two document types are *Working Drafts* and *Candidate Specifications*.
+Theare are three document types: *Proposals*, *Working Drafts*, and *Candidate Specifications*.
 
-1. When a community member wishes to submit a new document for comment they will submit a Pull Request to the [RFCs repo](https://github.com/interledger/rfcs/) which adds a new Markdown file. The file MUST follow the naming convention `0000-title`, where title is a lower case title with spaces replaced by hyphens (`-`), and have front-matter (similar to GitHub pages rendered from MarkDown) specifying at least a `title` and `draft` (an integer, starting at 1 and incrmenting with each revision of the RFC).
+### Proposals
+A _Proposal_ is an RFC document that proposes some new feature, protocol or idea related to Interledger. The criteria for getting a proposal document merged is minimal: project maintainers will review the document to ensure style conformance and applicability to Interledger standards, but will otherwise not require editorial fixes before allowing a proposal to be published.
+
+#### Publishing a Proposal
+To publish a new Proposal, submit a Pull Request to the [RFC Proposals repo](https://github.com/interledger/rfcs/proposals) with a new Markdown file. The file MUST follow the naming convention `0000-title`, where title is a lower case title with spaces replaced by hyphens (`-`), and have front-matter (similar to GitHub pages rendered from MarkDown) specifying at least a `title`, `type`, and `draft` (an integer, starting at 1 and incrementing with each revision of the RFC). The `type` MUST have the value `proposal`.
 
 Example:
-```
----
+```markdown
+\---
 title: The Interledger Architecture
+type: proposal
 draft: 1
----
+\---
 # Interledger Architecture
 
 Lorum Ipsum etc
 ```
 
-2. One of the project maintainers will review the submission and assign an RFC number to the document, making a follow up commit to the PR which renames the file and folder as appropriate.
+### Working Drafts 
+A _Working Draft_ is an RFC that has gained significant consensus support from the Interledger community. Working Drafts differ from Proposals in that edits to the document undergo significant community review and are not merged until there is broad consenus among the editors of the RFC, and the broader Interledger community.
```

Comment:
I don't think there is a need for a working draft to have "significant consensus".

I like @matdehaast 's suggestions to how we define a WD. It is stable enough for people to experiment but not endorsed by the community.

It should be possible to have two WDs that address the same problem in different ways and ultimately either get merged into one or the consensus of the community is to adopt one and not the other meaning the WD becomes a candidate spec.

Reply:
Agreed - this is addressed in the latest commit.